tests/e2e/testdata/scenarios/direct/mutable_but_unreadable/SCENARIO_README.md

```diff
@@ -0,0 +1,7 @@
+For the LoggingLogMetric resource the metadata fields are:
+
+```yaml
+    cnrm.cloud.google.com/mutable-but-unreadable-fields: '{"spec":{"metricDescriptor":{"launchStage":"EARLY_ACCESS","metadata":{"ingestDelay":"1s","samplePeriod":"1s"}}}}'
+```
+
+What we want to see is that if we update the spec (the user intent) of our LoggingLogMetric resource WITH the same values for our mutable but unreadable fields, that update DOES NOT cause an underlying cloud provider API call. That is, there is no http log for the apply.
```

Comment:
I may not have enough context about this test, but from this README `there is no http log for the apply`, I expect to not see a POST request. But I still see there's one POST in _http00.log. Could you help me understand the expectation of this test?

Reply:
I think the second object corresponds to _http01.log, and there is no _http01.log.  Maybe we should write an empty golden file, it's probably more an accident that we don't than a deliberate design decision

Reply:
Yes maybe the README needs to spell out that we won't have an `_http01.log` file but I can come back here when I add / activate the tests for the direct controller.

> that we won't have an `_http01.log` file

NOTE that having GET requests to begin with would be a-ok because just as long as we don't see a POST/ PUT in that log that we wouldn't expect!